Godeps/Godeps.json

```diff
@@ -377,6 +406,10 @@
 			"ImportPath": "k8s.io/gengo/args",
 			"Rev": "75356185a9af8f0464efa792e2e9508d5b4be83c"
 		},
+		{
+			"ImportPath": "k8s.io/gengo/examples/set-gen/sets",
```

Comment:
this surprised me... what pulled this in?

Reply:
Has nothing to do with the PR, but the godeps were out of date:
```
$ git grep k8s.io/gengo/examples/set-gen/sets                                                                                                                  Godeps/Godeps.json:                     "ImportPath": "k8s.io/gengo/examples/set-gen/sets",
pkg/generators/extension.go:    "k8s.io/gengo/examples/set-gen/sets"
pkg/generators/extension_test.go:       "k8s.io/gengo/examples/set-gen/sets"
```